test/e2e-common.sh

```diff
@@ -119,6 +119,44 @@ function run_yaml_tests() {
 }
 
 function install_pipeline_crd() {
+  echo ">> install operator"
+  mypath=$(pwd)
+
+  echo ">> check current path: $mypath"
+
+  echo ">> check gopath $GOPATH"
+  echo $GOPATH
+
+  echo ">> check git version"
+  git version
+
+  cd $GOPATH
+  mkdir -p src/github.com/tektoncd
+  cd src/github.com/tektoncd
+  git clone https://github.com/tektoncd/operator.git
```

Comment:
It seems a bit brittle to me that pipelines' integration tests must install the Operator from HEAD (actually from a PR?) to be able to perform any tests. If the Operator repo has a breakage, development of Pipelines will be broken.

What does taking this dependency buy us? The PR description sounds like this will emit data about the performance of the Operator (does that include the performance of the Pipelines components themselves?), but since this is only testing committed changes to the Operator any performance regression we find would already be landed in Operator, so it's a bit late. Wouldn't Operator performance tests be better placed in the Operator repo, where they can test regressions before PRs are merged?

Perhaps I'm missing some context, in which case filling in the PR description template should be able to answer these questions.